content/en/admin/install.md

```diff
@@ -84,14 +87,8 @@ git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-b
 Once this is done, we can install the correct Ruby version:
 
 ```bash
-RUBY_CONFIGURE_OPTS=--with-jemalloc rbenv install 3.2.3
-rbenv global 3.2.3
-```
-
-We’ll also need to install the bundler:
-
-```bash
-gem install bundler --no-document
+RUBY_CONFIGURE_OPTS=--with-jemalloc rbenv install 3.3.5
```

Comment:
Do we have a way to use `.ruby-version` here, rather than hardcoding this in the doc? This would require doing this after the `git clone`, but it would make more sense?

Reply:
Omitting the version should use the one defined in `.ruby-version` afaik

Reply:
Would skip the `rbenv global` as well.

Reply:
Why? ruby-install finishes with a message that tells you do this, so I would imagine many would do it anyway. Also, it might come in handy if an admin tries anything with ruby that relates to mastodon. And even though it says `global`, it is really limited to the `mastodon` user here.

Reply:
Oh, just to avoid having any reference to a hardcoded version. If running in the code directory, rbenv will pick up the `.ruby-version`

Reply:
Ah, yes that makes sense. I will restructure so that we can pick up `.ruby-version` and then remove the `rbenv global` call.

Reply:
I restructured the instructions (and validated that they work as intended again). Not only do we get rid of the concrete version number this way, I think it further simplifies the whole process (less user switching).